src/Select.js

```diff
@@ -1038,48 +1038,54 @@ export default class Select extends Component<Props, State> {
       MultiValueRemove,
       SingleValue,
       Placeholder,
+      ValueSpacer,
     } = this.components;
     const { commonProps } = this;
-    const { isDisabled, isMulti, inputValue, placeholder } = this.props;
+    const { isDisabled, isMulti, inputValue, placeholder, options } = this.props;
     const { selectValue } = this.state;
 
     if (!this.hasValue()) {
       return inputValue ? null : (
-        <Placeholder {...commonProps} key="placeholder" isDisabled={isDisabled}>
-          {placeholder}
-        </Placeholder>
+        <ValueSpacer key="placeholder" values={{ placeholder, options }}>
```

Comment:
This gives a wrapper over any values selected (note: I only really thought about the single value case)

Reply:
Ideally it's an option for users to opt-into this behaviour.